lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -269,9 +289,9 @@ def main():
     new_path = module.params.get('new_path')
     dup_ok = module.params.get('dup_ok')
     if state == 'present' and not new_name and not new_path:
-        cert = module.params.get('cert')
-        key = module.params.get('key')
-        cert_chain = module.params.get('cert_chain')
+        cert, key, cert_chain = load_data(cert=module.params.get('cert'),
```

Comment:
There is one downside to this approach: if a user passes a nonexistent file path, this module will error out with some variation of `400: invalid certificate`. In that case, we need to make sure the error message surfaces the information about what certs were invalid. 

A way to check this might be if the "cert" is less than 200 characters, we can be certain it's supposed to be a file path since the server certs have to be at least 1024 bit. Over that, it's a maybe, but we should print out the cert that's supposedly "invalid" so people can tell what's wrong.